### DISCUSSION TITLE
FEEDBACK on Multiplayer

### DISCUSSION THREAD

**Discussion by KroeteDE** [ppy/osu#27010] [Ideas]
### Type

Game behaviour

### Bug description

you should get forced to playmode in multiplayer when u spectate for one map i always forget to click on back again and its so frustrating 

sorry that i post this feedback here

### Version

latest

### Logs

no

**Comment by bdach** [comment]
I don't understand what this is trying to say. Are you asking for the game to automatically "un-spectate" you after every map played?

**Comment by Qunas101** [comment]
maybe a button to spectate until the end of current map, idk. I understand what OP means, sometimes you forget and have to find the button to cancel at the last second (which is greyed out btw and it doesn't help). When you are in the lobby view, you kinda assume that you are going to play with everyone.
